siemens/questa.mk

```diff
@@ -158,9 +177,19 @@ $(DEP_DIR)/%.questa.o: $(SIM_LIB_DONE) | $(DEP_DIR) $(BLOG_DIR)
 	else echo "Unknown filetype: $(word 2,$^)"; echo "$^"; exit 1; fi; fi;
 	@touch $@
 
+AC_CMD = qverify -c -do $(AC_SCRIPT) -od $(AC_OUT_DIR) -modelsimini $(MS_INI)
```

Comment:
change `=` to `:=`

Reply:
Oh yeah I changed it to `=` because I thought it wasn't detecting a file change but I fixed it another way and forgot to change this back.
Thanks, good catch.